Neos.Media/Classes/Domain/ValueObject/Configuration/README.md

```diff
@@ -0,0 +1,45 @@
+# Configuration Value Objects
```

Comment:
This documentation should be moved to the documentation.

Reply:
Found it in the docs, can be removed?

Reply:
Well, theoretically, yes, but I thought that it would be a good idea for core devs to have some more information next to the code. The manual should provide user facing documentation and this is a hint for maintainers of the code. I like it, but if you insist, we can also remove it.